recipes/mosquitto/meta.yaml

```
@@ -0,0 +1,70 @@
+{% set name = "mosquitto" %}
+{% set version = "2.0.14" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://mosquitto.org/files/source/{{ name }}-{{ version }}.tar.gz
+  sha256: d0dde8fdb12caf6e2426b4f28081919a2fce3448773bdb8af0d3cd5fe5776925
+
+build:
+  number: 0
+  skip: True  # [osx]
+  run_exports:
+    - {{ pin_subpackage( 'mosquitto' ) }}
+  ignore_run_exports:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - cmake
+    - make
+    - libxslt  # [not win]
+    - ucrt  # [win]
+  host:
+    - openssl
```

```suggestion
  host:
    - openssl
  run:
    - ucrt  # [win]
```
You seem to be having trouble with this.

@nerohmot Do you need the docs/manpages? It seems that libxslt is only used to build the documentation which is why no libraries link to it.

You can disable the documentation by adding `-DDOCUMENTATION=OFF` to the CMake args and then remove libxslt from the requirements. This would solve the overdepending problem.

Related: https://github.com/eclipse/mosquitto/issues/2272

---

Thanx for the helping hand! 
Indeed I am struggling with this one :frowning_face: 
... still I don't understand why the libxslt must go ...
Anyway, will do a you proposed and then work from there (including the `-DDOCUMENTATION=OFF` arg tip)

---

In the mosquitto README, it says that libxslt is only required to build to documentation, so if you don't want/need the docs in this package, then we can remove it from the build process. Do you want/need the docs?

When you put libxslt in the host section, it tells conda that this library should be linked against, so when the link inspector looks at the binaries in the libmosquitto package and doesn't find any links to libxslt it creates a warning because that could mean that libxslt is being statically linked (which is not allowed on conda forge). In this specific case, libxslt is not linked because it is not needed by the libmosquitto binaries.